x-pack/plugins/apm/public/components/routing/apm_route_config.tsx

```diff
@@ -53,6 +53,12 @@ const apmRoutes = route([
         }),
       }),
     ]),
+    defaults: {
+      query: {
+        rangeFrom: 'now-15m',
+        rangeTo: 'now',
+      },
+    },
```

Comment:
is this needed?

Reply:
Is, the ranges are not required when we call this link, andor it's not provided we should add use the default instead